atlasdb-config/src/test/java/com/palantir/atlasdb/factory/LeadersTest.java

```diff
@@ -116,19 +115,6 @@ public void createProxyAndLocalListCreatesSingletonListIfNoRemoteAddressesProvid
         verifyNoMoreInteractions(localAcceptor);
     }
 
-    @Test(expected = IllegalArgumentException.class)
```

Comment:
This is basically testing some code in Feign, so I'll say not worth it.

Reply:
Yes. This looks like it originally sought to test that the AtlasDbFeignTargetFactory which doesn't exist anymore wouldn't do silly things

Reply:
(When we had a lot of custom code there, but now we don't)